lib/go/test/market_test.go

```diff
@@ -46,10 +47,11 @@ func TestMarketDeployment(t *testing.T) {
 	// Should be able to deploy the MetadataViews contract
 	// as a new account with no keys.
 	metadataViewsCode, _ := DownloadFile(MetadataViewsContractsBaseURL + MetadataViewsInterfaceFile)
+	parsedMetadataContract := strings.Replace(string(metadataViewsCode), MetadataReplaceAddress, "0x"+emulatorFTAddress, 1)
```

Comment:
The downloaded file at [MetadataViews](https://raw.githubusercontent.com/onflow/flow-nft/master/contracts/MetadataViews.cdc) needed some formatting to parse the emulator FT address